Minor tweak to burst limits (#8656)
braze-inc/braze-docs: _docs/_api/endpoints/user_data/post_user_track.md

## Monthly Active Users CY 24-25
For customers who have purchased Monthly Active Users - CY 24-25, Braze manages different rate limits on its `/users/track` endpoint:
- Hourly rate limits are set according to the expected data ingestion activity on your account, which may correspond to the number of monthly active users you have purchased, industry, seasonality, or other factors.

> - In addition to the hourly limit, Braze enforces a burst limit on the number of requests sent per three second interval.

Reviewer: ```suggestion
- In addition to the hourly limit, Braze enforces a burst limit on the number of requests that can be sent every three seconds.
```

> - In addition to the limit on requests per hour, Braze also imposes a burst limit on the number of requests allowed per three second interval.

Reviewer: ```suggestion
- In addition to the hourly limit, Braze enforces a burst limit of  X requests every three seconds.
```

or if a specific number can't be specified:

```suggestion
- In addition to the hourly limit, Braze enforces a burst limit on the number of requests sent per three second interval.
```

@colleenkerr , do you know if there's a specific burst limit number?

Author: The limit is defined per customer so unfortunately we can't publish it here. Because of that it's only available in the Braze dashboard (mentioned in the paragraph below this)
